server/routes/allCompletedReports/allCompletedReports.ts

```diff
@@ -31,7 +31,7 @@ export default class AllCompletedReportsRoutes {
   ): Promise<void> =>
     res.render(`pages/allCompletedReports`, {
       allCompletedReports: results,
-      filter,
+      filter: filter == null ? [{ status: 'SCHEDULED' }] : filter,
```

Comment:
I thought that if no filter was provided it meant that all statuses should be selected? Not just SCHEDULED